config/jetscape_user_MUSIC.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+
+<jetscape>
+
+  <vlevel> 0 </vlevel>
+  <nEvents> 1 </nEvents>
+
+  <JetScapeWriterAscii> on </JetScapeWriterAscii>
+
+    <!-- Inital State Module  -->
+    <IS>
+        <grid_max_x> 15 </grid_max_x>
+        <grid_max_y> 15 </grid_max_y>
+        <grid_max_z> 0 </grid_max_z>
+        <grid_step_x> 0.2 </grid_step_x>
+        <grid_step_y> 0.2 </grid_step_y>
+        <Trento>
+        <PhysicsInputs  projectile='Au'
+                target='Au'
+                sqrts='200'
+                cross-section='4.2'
+                normalization='5.73'>
+        </PhysicsInputs>
+        <CutInputs  centrality-low='0'
+              centrality-high='10'>
+        </CutInputs>
+        <TransInputs  reduced-thickness='0.0'
+                fluctuation="1.2"
+                nucleon-width="0.9"
+                nucleon-min-dist="1.2">
+        </TransInputs>
+        <LongiInputs  mean-coeff="1.0"
+                std-coeff="3.0"
+                skew-coeff="0.0"
+                skew-type="1"
+                jacobian="0.8">
+        </LongiInputs>
+        </Trento>
+    </IS>
+
+  <!-- Hard Process -->
+  <Hard>
+    <PythiaGun>
+      <pTHatMin>50</pTHatMin>
+      <pTHatMax>70</pTHatMax>
+      <eCM>5020</eCM>
```

Comment:
Hi Chun in any case if this is the example xml for RHIC we should change pThat bin and sqrts in. Pythia too, maybe 20-30 and 200 ...

Reply:
Updated.